tests/unit_tests/test_utils.py

```diff
@@ -0,0 +1,87 @@
+import os
+import tempfile
+import pytest
+import logging
+
+
+def get_last_line(file_path):
```

Comment:
A bit unfortunate you cant import the actual function

Reply:
I will handle it in another PR.